slasher/db/kv/kv.go

```diff
@@ -25,6 +25,14 @@ type Store struct {
 	spanCacheEnabled bool
 }
 
+// EpochStore defines an implementation of the slasher data access interface
+// using byte array as data source to extract and put validator spans into.
+type EpochStore struct {
```

Comment:
Does this need to be a struct? Look at the go-bitfield implementation at how it forms the interface around a single type.

Reply:
replaced. looks much better now